contracts/ColonyNetwork.sol

```diff
@@ -107,9 +105,21 @@ contract ColonyNetwork is ColonyNetworkStorage {
     return reputationRootHashNNodes;
   }
 
-  function createColony(bytes32 _name, address _tokenAddress) public
-  colonyKeyUnique(_name)
+  function createMetaColony(address _tokenAddress) public 
+  auth
   {
+    // Add the root global skill
```

Comment:
Let's put a `require(metaColony==0)` in here. Accidentally running this function twice would be... bad.